updateContacts.html

```diff
@@ -41,5 +41,8 @@
 				<span id="loginResult"></span>
 			</div>
 		</div>
+		<script type="text/javascript">
+			window.onload = contactsAutoFill();
+		</script>
```

Comment:
Needs to be at the end of the body in order to AutoFill properly. Otherwise, will attempt to autofill elements that don't exist. @CathyChian @cereusware